app/src/main/java/com/gdg/pune/devfest19/ui/fragment/SpeakersFragment.kt

```diff
@@ -27,7 +27,7 @@ class SpeakersFragment : AbstractRecyclerFragment() {
             )
 
             override fun onBindViewHolder(viewHolder: SpeakerViewHolder, position: Int, model: Speaker) {
-                viewHolder.bind(model, null)
+                viewHolder.bind(model)
```

Comment:
Actually, When we are passing document id it should be pass from intent. That intent should be in adapter class not im ViewHolder class. We just have to set listener in ViewHolder class. Refer code of `ScheduleFragment`.